### PR TITLE
[12.0][IMP] project_timeline - Enable timeline view with subtasks

### PR DESCRIPTION
**Pull request by emagdalenaC2i**

### CODE REVIEW

File: project_timeline/views/project_task_view.xml [hunk 1]
```diff
@@ -44,6 +44,10 @@
         <field name="view_mode">kanban,tree,form,calendar,timeline,pivot,graph</field>
     </record>
 
+    <record id="project_task_action_sub_task" model="ir.actions.act_window">
+        <field name="view_mode">kanban,tree,form,calendar,pivot,graph,activity</field>
```
Comment (pedrobaeza) [inline project_timeline/views/project_task_view.xml]: Where is timeline?

**Comment by emagdalenaC2i** [comment]
Runbot and travis are green but I get this error when I try to test it in runbot
The style compilation failed, see the error below. Your recent actions may be the cause, please try reverting the changes you made.

Error: Invalid CSS after "...    background:": expected expression (e.g. 1px, bold), was "@progress-bar-succe"
        on line 9683 of stdin
>>         background: @progress-bar-success-bg;
   --------------------^
This error occured while compiling the bundle 'web.assets_backend' containing:
    - /web/static/lib/bootstrap/scss/_functions.scss
    - /web/static/lib/bootstrap/scss/_mixins.scss
    - /web/static/src/scss/utils.scss
    - /web/static/src/scss/bs_mixins_overrides.scss
    - /web/static/src/scss/primary_variables.scss
    - /web_editor/static/src/scss/web_editor.variables.scss
    - /mail/static/src/scss/variables.scss
    - /portal/static/src/scss/primary_variables.scss
    - /web/static/src/scss/secondary_variables.scss
    - /web/static/src/scss/bootstrap_overridden.scss
    - /web_editor/static/src/scss/bootstrap_overridden.scss
    - /web/static/src/scss/import_bootstrap_part1.scss
    - /web/static/src/scss/bs_mixins_overrides.scss
    - /web/static/src/scss/import_bootstrap_part2.scss
    - /web/static/src/scss/bootstrap_review.scss
    - /web/static/src/scss/webclient_extra.scss
    - /web/static/src/scss/webclient_layout.scss
    - /web/static/src/scss/webclient.scss
    - /web/static/src/scss/domain_selector.scss
    - /web/static/src/scss/model_field_selector.scss
    - /web/static/src/scss/progress_bar.scss
    - /web/static/src/scss/dropdown.scss
    - /web/static/src/scss/dropdown_extra.scss
    - /web/static/src/scss/tooltip.scss
    - /web/static/src/scss/debug_manager.scss
    - /web/static/src/scss/control_panel.scss
    - /web/static/src/scss/fields.scss
    - /web/static/src/scss/fields_extra.scss
    - /web/static/src/scss/views.scss
    - /web/static/src/scss/pivot_view.scss
    - /web/static/src/scss/graph_view.scss
    - /web/static/src/scss/form_view.scss
    - /web/static/src/scss/form_view_extra.scss
    - /web/static/src/scss/list_view.scss
    - /web/static/src/scss/list_view_extra.scss
    - /web/static/src/scss/kanban_dashboard.scss
    - /web/static/src/scss/kanban_examples_dialog.scss
    - /web/static/src/scss/kanban_column_progressbar.scss
    - /web/static/src/scss/kanban_view.scss
    - /web/static/src/scss/kanban_view_mobile.scss
    - /web/static/src/scss/web_calendar.scss
    - /web/static/src/scss/search_view.scss
    - /web/static/src/scss/search_view_mobile.scss
    - /web/static/src/scss/dropdown_menu.scss
    - /web/static/src/scss/search_view_extra.scss
    - /web/static/src/scss/data_export.scss
    - /base/static/src/scss/onboarding.scss
    - /web/static/src/scss/base_settings.scss
    - /web/static/src/scss/report_backend.scss
    - /base_import/static/src/scss/base_import.scss
    - /web_diagram/static/src/scss/diagram_view.scss
    - /web_editor/static/src/scss/web_editor.common.scss
    - /web_editor/static/src/scss/web_editor.backend.scss
    - /mail/static/src/scss/announcement.scss
    - /mail/static/src/scss/discuss.scss
    - /mail/static/src/scss/abstract_thread_window.scss
    - /mail/static/src/scss/thread_window.scss
    - /mail/static/src/scss/composer.scss
    - /mail/static/src/scss/chatter.scss
    - /mail/static/src/scss/followers.scss
    - /mail/static/src/scss/thread.scss
    - /mail/static/src/scss/systray.scss
    - /mail/static/src/scss/mail_activity.scss
    - /mail/static/src/scss/activity_view.scss
    - /mail/static/src/scss/kanban_view.scss
    - /mail/static/src/scss/attachment_box.scss
    - /hr/static/src/scss/hr.scss
    - /partner_autocomplete/static/src/scss/partner_autocomplete.scss
    - /project/static/src/scss/project_dashboard.scss
    - /project_timeline/static/src/less/project_timeline.less
    - /project_timeline_hr_timesheet/static/src/scss/project_timeline_hr_timesheet.scss
    - /web_settings_dashboard/static/src/scss/dashboard.scss

@pedrobaeza It also happend a few days ago testing #487

**Comment by pedrobaeza** [comment]
Well, I can't say, but this is working on my side, and the other PR is for other version (10.0), so no possible link. Maybe runbot was broken in that moment. There has been a maintenance this weekend. Try on main branch to see if the error is there. If not, your changes are provoking that.

**Comment by emagdalenaC2i** [comment]
close this PR, see #495

**Pull request closed without merging**